protocols/gossipsub/src/behaviour.rs

```diff
@@ -1528,7 +1528,7 @@ where
                 )
                 .is_err()
             {
-                error!("Failed to send graft. Message too large");
+                error!("Failed to send PRUNE. Message too large");
             }
```

Comment:
Is "Message to large" really the only failure mode here? That is not obvious from just checking `is_err`?

Reply:
You are right. 💡 I'll update this.